pages/common/ppmflash.md

```diff
@@ -0,0 +1,12 @@
+# ppmflash
+
+> Brighten a PPM image file.
+> More information: <https://netpbm.sourceforge.net/doc/ppmflash.html>.
+
+- Brighten a PPM image by the fraction `falshfactor`:
+
+`ppmflash {{flashfactor}} {{path/to/file.ppm}}`
```

Comment:
Please clarify whether this change happens in-place or is printed to stdout

Reply:
A new PPM image file would be generated.
I have updated the ppmflash.md file, please check it.